sql-statements/sql-statement-explain-analyze.md

```diff
@@ -82,15 +82,176 @@ EXPLAIN ANALYZE SELECT * FROM t1;
+```
+
+- `Inner`: the execution information of inner worker:
+    - `total`: the total time-consuming by the inner worker.
+    - `concurrency`: the number of inner workers.
+    - `task`: The total number of tasks processed by the inner worker.
+    - `construct`: the prepare time before the inner worker reads the inner table rows.
+    - `fetch`: The total time-consuming of inner worker to read inner table rows.
+    - `Build`: The total time-consuming of inner worker to construct the hash map of the outer table rows.
+    - `join`:  The total time-consuming of inner worker to do join with the inner table rows and the hash map of outer table rows.
+
+### HashJoin
+
+The `HashJoin` operator has an inner worker, an outer worker and N join workers. The specific execution process is as follows:
+
+1. The inner worker reads inner table rows and constructs a hash map.
+2. The outer worker reads the outer table rows, then wraps it into a task and sends it to the join worker.
+3. Wait for the completion of the hash map construction in step 1.
+4. The join worker uses the outer table rows and hash map in the task to do join, and then sends the join result to the result channel.
+5. The main thread of `HashJoin` receives the join result from the result channel.
+
+The `HashJoin` operator contains the following execution information:
+
+```
+build_hash_table:{total:146.071334ms, fetch:110.338509ms, build:35.732825ms}, probe:{concurrency:5, total:857.162518ms, max:171.48271ms, probe:125.341665ms, fetch:731.820853ms}
+```
+
+- `build_hash_table`: Read the data of the inner table and construct the execution information of the hash map:
+    - `total`: total time spent.
+    - `fetch`: The total time spent reading inner table data.
+    - `build`: The total time spent constructing a hash map.
+- `probe`: execution information of join worker:
+    - `concurrency`: the number of join workers.
+    - `total`: the total time consumed by all join workers.
+    - `max`: The maximum time for a single join worker to execute.
+    - `probe`: The total time consumed for joining with outer table rows and hash map.
+    - `fetch`: The total time that the join worker waits to read the outer table rows data.
+
+## Other execution info
+
+### lock_keys execution information
+
+When executing a DML statement in a pessimistic transaction, the execution information of the operator may also include the execution information of `lock_keys`. For example:
+
+```
+lock_keys: {time:94.096168ms, region:6, keys:8, lock_rpc:274.503214ms, rpc_count:6}
+```
+
+- `time`: The total time to execute the `lock_keys` operation.
+- `region`: The number of regions involved in executing the `lock_keys` operation.
+- `keys`: The number of `Key` that need `Lock`.
+- `lock_rpc`: The total time spent sending a RPC of type `Lock` to TiKV. Because multiple RPC requests can be sent in parallel, the total RPC time-consuming may be greater than the total time-consuming `lock_keys` operation.
```

Comment:
```suggestion
- `lock_rpc`: The total time spent sending an RPC request of the `Lock` type to TiKV. Because multiple RPC requests can be sent in parallel, the total RPC time consumption might be greater than the total time consumption of the `lock_keys` operation.
```